TestProjects/PerceptionURP/Assets/Scenes/SampleScene.unity

```diff
@@ -98,7 +98,7 @@ LightmapSettings:
     m_TrainingDataDestination: TrainingData
     m_LightProbeSampleCountMultiplier: 4
   m_LightingDataAsset: {fileID: 0}
-  m_UseShadowmask: 0
+  m_UseShadowmask: 1
```

Comment:
Not related to this PR, but I noticed the URP test project has baked GI and lighting auto-generate turned on. Should we perhaps turn these off?